vitest.config.mts

```diff
@@ -0,0 +1,30 @@
+import { fileURLToPath, URL } from 'node:url'
+import { defineConfig, configDefaults } from 'vitest/config'
+import react from '@vitejs/plugin-react'
+
+export default defineConfig({
+	plugins: [react()],
+	test: {
+		globals: true,
+		environment: 'jsdom',
+		exclude: [...configDefaults.exclude, 'src/__tests__/e2e'],
+		setupFiles: ['dotenv/config', '.test/setup-vitest.js'],
+	},
+	resolve: {
```

Comment:
Let's move to using the plugin [vite-tsconfig-paths](https://github.com/aleclarson/vite-tsconfig-paths), [as recommended by the docs](https://vitest.dev/guide/common-errors#cannot-find-module-relative-path), so we don't have to worry about updating these when we add alias's.

Reply:
Hmmm, let me pull down this PR and play with it for 15 mins. Maybe the second try will work. Great job on this helper function btw.

Reply:
Huh, I was able to pass the same number of test files, 70, with using vite-tsconfig-paths with no options:

`plugins: [react(), tsconfigPaths()],`

I made sure that the resolve code was commented out, and [ran with no cache.](https://vitest.dev/config/#cache)

Reply:
Oh weird! I was trying to use it at the beginning of the migration, so I wonder if I was confusing another error with it not working. Thanks for getting it up and running! Much better :)

Reply:
Computers 🤷🏻‍♂️